h/zssLogging.h

```diff
@@ -52,529 +52,483 @@ bool isLogLevelValid(int level);
 #define ZSS_LOG_PATH_TO_SERVER_MSG           ZSS_LOG_PATH_TO_SERVER_MSG_ID" "ZSS_LOG_PATH_TO_SERVER_MSG_TEXT
 
 #ifndef ZSS_LOG_SIG_IGNORE_MSG_ID
-#define ZSS_LOG_SIG_IGNORE_MSG_ID            LOG_MSG_PRFX"1032W"
+#define ZSS_LOG_SIG_IGNORE_MSG_ID            ZSS_LOG_MSG_PRFX"1032W"
 #endif
-#define ZSS_LOG_SIG_IGNORE_MSG_TEXT          "warning: sigignore(SIGPIPE)=%d, errno=%d\n"
+#define ZSS_LOG_SIG_IGNORE_MSG_TEXT          "Sigignore(SIGPIPE)=%d, errno=%d\n"
 #define ZSS_LOG_SIG_IGNORE_MSG               ZSS_LOG_SIG_IGNORE_MSG_ID" "ZSS_LOG_SIG_IGNORE_MSG_TEXT
 
 #ifndef ZSS_LOG_SERVER_CONFIG_MSG_ID
-#define ZSS_LOG_SERVER_CONFIG_MSG_ID         LOG_MSG_PRFX"1033I"
+#define ZSS_LOG_SERVER_CONFIG_MSG_ID         ZSS_LOG_MSG_PRFX"1033I"
 #endif
 #define ZSS_LOG_SERVER_CONFIG_MSG_TEXT       "Server config file=%s\n"
 #define ZSS_LOG_SERVER_CONFIG_MSG            ZSS_LOG_SERVER_CONFIG_MSG_ID" "ZSS_LOG_SERVER_CONFIG_MSG_TEXT
 
 #ifndef ZSS_LOG_SERVER_STARTUP_MSG_ID
-#define ZSS_LOG_SERVER_STARTUP_MSG_ID        LOG_MSG_PRFX"1034W"
+#define ZSS_LOG_SERVER_STARTUP_MSG_ID        ZSS_LOG_MSG_PRFX"1034W"
 #endif
-#define ZSS_LOG_SERVER_STARTUP_MSG_TEXT      "server startup problem, address %s not valid\n"
+#define ZSS_LOG_SERVER_STARTUP_MSG_TEXT      "Server startup problem, address %s not valid\n"
 #define ZSS_LOG_SERVER_STARTUP_MSG           ZSS_LOG_SERVER_STARTUP_MSG_ID" "ZSS_LOG_SERVER_STARTUP_MSG_TEXT
 
 #ifndef ZSS_LOG_ZSS_SETTINGS_MSG_ID
-#define ZSS_LOG_ZSS_SETTINGS_MSG_ID          LOG_MSG_PRFX"1035I"
+#define ZSS_LOG_ZSS_SETTINGS_MSG_ID          ZSS_LOG_MSG_PRFX"1035I"
 #endif
 #define ZSS_LOG_ZSS_SETTINGS_MSG_TEXT        "ZSS server settings: address=%s, port=%d\n"
 #define ZSS_LOG_ZSS_SETTINGS_MSG             ZSS_LOG_ZSS_SETTINGS_MSG_ID" "ZSS_LOG_ZSS_SETTINGS_MSG_TEXT
 
 #ifndef ZSS_LOG_ZSS_STARTUP_MSG_ID
-#define ZSS_LOG_ZSS_STARTUP_MSG_ID           LOG_MSG_PRFX"1036W"
+#define ZSS_LOG_ZSS_STARTUP_MSG_ID           ZSS_LOG_MSG_PRFX"1036W"
 #endif
 #define ZSS_LOG_ZSS_STARTUP_MSG_TEXT         "Server startup problem ret=%d reason=0x%x\n"
 #define ZSS_LOG_ZSS_STARTUP_MSG              ZSS_LOG_ZSS_STARTUP_MSG_ID" "ZSS_LOG_ZSS_STARTUP_MSG_TEXT
 
 #ifndef ZSS_LOG_PORT_OCCUP_MSG_ID
-#define ZSS_LOG_PORT_OCCUP_MSG_ID            LOG_MSG_PRFX"1037W"
+#define ZSS_LOG_PORT_OCCUP_MSG_ID            ZSS_LOG_MSG_PRFX"1037W"
 #endif
 #define ZSS_LOG_PORT_OCCUP_MSG_TEXT          "This is usually due to the server port (%d) already being occupied. Is ZSS running twice?\n"
 #define ZSS_LOG_PORT_OCCUP_MSG               ZSS_LOG_PORT_OCCUP_MSG_ID" "ZSS_LOG_PORT_OCCUP_MSG_TEXT
 
 /* MVD Server (Datasets) */
 
 #ifndef ZSS_LOG_DS_NAME_HLQ_MSG_ID
-#define ZSS_LOG_DS_NAME_HLQ_MSG_ID           LOG_MSG_PRFX"1038I"
+#define ZSS_LOG_DS_NAME_HLQ_MSG_ID           ZSS_LOG_MSG_PRFX"1038I"
 #endif
-#define ZSS_LOG_DS_NAME_HLQ_MSG_TEXT         "l1=%s\n"
+#define ZSS_LOG_DS_NAME_HLQ_MSG_TEXT         "L1=%s\n"
```

Comment:
@DivergentEuropeans tbh, this also seems like a debug message, what do you think?

(I haven't reviewed every single text)

Reply:
I agree